other/bootstrap_daemon/tox_bootstrap.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Tox DHT Bootstrap Daemon
+After=network.target
+
+[Service]
+Type=forking
+PermissionsStartOnly=true
+ExecStartPre=/usr/bin/touch /var/run/tox-bootstrapd.pid
```

Comment:
Now it's starting in background and leaves PID file. Systemd knows PID, and can kill process by it's pid on "stop" signal.

If where is no PID file - what kind of mechanism must be used  to stop the process by init system?

Systemd can run process, that is not daemonized by itself, by sending to background some subprocess of systemd (i don't know, how it works exactly), which  staying with controlled process in foreground. Like virtual console.